Homeworks/Homework SQL/SQLQuery-Homework.sql

```diff
@@ -0,0 +1,339 @@
+VALUES ('andrei', 'stan', 'as@gmail.com');
+INSERT INTO Author
+VALUES ('diana', 'ciocan', 'ac@gmail.com');
+INSERT INTO Author
+VALUES ('teodor', 'nistor', 'tn@gmail.com');
+INSERT INTO Author
+VALUES ('stefania', 'iancu', 'si@gmail.com');
+
+SELECT *
+FROM Author;
+
+--crime, fantasy, horror, adventure
+INSERT INTO TYPE
+VALUES ('crime');
+INSERT INTO TYPE
+VALUES ('fantasy');
+INSERT INTO TYPE
+VALUES ('horror');
+INSERT INTO TYPE
+VALUES ('adventure');
+
+SELECT *
+FROM TYPE;
+
+INSERT INTO Books
+VALUES ('Book1',1,1);
+INSERT INTO Books
+VALUES ('Book2',1,2);
+INSERT INTO Books
+VALUES ('Book3',2,3);
+INSERT INTO Books
+VALUES ('Book4',3,4);
+INSERT INTO Books
+VALUES ('Book5',4,2);
+INSERT INTO Books
+VALUES ('Book6',5,4);
+INSERT INTO Books
+VALUES ('Book7',4,3);
+INSERT INTO Books
+VALUES ('Book8',3,2);
+INSERT INTO Books
+VALUES ('Book9',2,1);
+
+SELECT *
+FROM Books;
+
+INSERT INTO Libraries
+VALUES ('Library 1','Bv. Independetei','Iasi');
+INSERT INTO Libraries
+VALUES ('Library 2','Bv. Eroilor','Bucuresti');
+INSERT INTO Libraries
+VALUES ('Library 3','Strada Brancusi','Brasov');
+INSERT INTO Libraries
+VALUES ('Library 4','Bv. Unirii','Bucuresti');
+INSERT INTO Libraries
+VALUES ('Library 5','Strada Stefan Cel Mare','Iasi');
+INSERT INTO Libraries
+VALUES ('Library 6','Bv. Socola','Iasi');
+
+-- Forgot to change library number
+DELETE FROM Libraries
+WHERE Id=6;
+
+SELECT * 
+FROM Libraries;
+
+INSERT INTO IsOnHoliday
+VALUES ('True');
+INSERT INTO IsOnHoliday
+VALUES ('False');
+
+SELECT *
+FROM IsOnHoliday;
+
+INSERT INTO Librarians
+VALUES ('Gustav','2022-01-01',1,1);
+INSERT INTO Librarians
+VALUES ('Robert','2010-01-01',7,2);
+INSERT INTO Librarians
+VALUES ('Giulia','2012-01-01',8,1);
+INSERT INTO Librarians
+VALUES ('Maxi','2019-01-01',9,2);
+INSERT INTO Librarians
+VALUES ('Vlad','2013-01-01',10,2);
+INSERT INTO Librarians
+VALUES ('Dana','2023-01-01',1,2);
+INSERT INTO Librarians
+VALUES ('Mihai','2007-01-01',11,1);
+INSERT INTO Librarians
+VALUES ('Ionut','2019-01-01',8,2);
+
+INSERT INTO Sales
+VALUES(13,1,1);
+INSERT INTO Sales
+VALUES(33,1,1);
+INSERT INTO Sales
+VALUES(312,7,2);
+INSERT INTO Sales
+VALUES(112,8,3);
+INSERT INTO Sales
+VALUES(53,9,4);
+INSERT INTO Sales
+VALUES(441,10,5);
+INSERT INTO Sales
+VALUES(444,11,6);
+INSERT INTO Sales
+VALUES(44,8,7);
+INSERT INTO Sales
+VALUES(124,9,8);
+INSERT INTO Sales
+VALUES(22,1,9);
+INSERT INTO Sales
+VALUES(44,7,10);
+INSERT INTO Sales
+VALUES(121,10,11);
+
+INSERT INTO Books_Sales
+VALUES (1,1);
+
+INSERT INTO Books_Sales
+VALUES (2,2);
+INSERT INTO Books_Sales
+VALUES (3,3);
+INSERT INTO Books_Sales
+VALUES (4,4);
+INSERT INTO Books_Sales
+VALUES (5,5);
+INSERT INTO Books_Sales
+VALUES (5,6);
+INSERT INTO Books_Sales
+VALUES (4,7);
+INSERT INTO Books_Sales
+VALUES (3,8);
+INSERT INTO Books_Sales
+VALUES (2,9);
+INSERT INTO Books_Sales
+VALUES (1,10);
+INSERT INTO Books_Sales
+VALUES (2,11);
+INSERT INTO Books_Sales
+VALUES (4,12);
+
+INSERT INTO Books_Libraries
```

Comment:
Same here. There are only 6 libraries added in the database above and yet you use LibraryId greater than 6. It doesn't work when running the code on a "clean" database.